packages/admin/src/components/DitoRoot.vue

```diff
@@ -158,6 +158,7 @@ export default DitoComponent.component('dito-root', {
 
     async login() {
       this.allowLogin = true
+      const { extraComponents, redirectAfterLogin } = this.options.login || {}
```

Comment:
I think I prefer `additionalComponents`. We also have `additionalProperties` in JSON schema and such.